docs/source/overview.rst

```diff
@@ -195,7 +195,7 @@ dimensionality of the referenced data.
 		tag = block.create_tag('interesting part', 'nix.roi', position)
 		tag.extent = extent
 		# finally, add the referenced data to this tag
-		tag.references.add(data)
+		tag.references.extend(data)
```

Comment:
not sure right now, but this should be ```append ```. ```extend``` should also work but for single entries append might be more appropriate